File: Hahn Softwareentwicklung/Models/WorkerModel.cs
namespace Hahn_Softwareentwicklung.Models
{
    public class WorkerModel
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
        public string Password { get; set; }
        public string Position { get; set; }
        public decimal Salary { get; set; }

        public WorkerModel(Guid id,string name, string email, string phoneNumber, string position, decimal salary)
        {
            Id = id;
            Name = name;
            Email = email;
            PhoneNumber = phoneNumber;
            Password = name + phoneNumber.ToString();
            Position = position;
            Salary = salary;
        }

    }
}

File: Angular/Hahn_Softwareentwicklung/src/app/models/worker.model.ts
export class WorkerModel {
    id: string;
    name: string;
    email: string;
    phoneNumber: string;
    password: string;
    position: string;
    salary: number;

    constructor(id:string, name:string, email:string, phoneNumber:string, password:string, position:string, salary:number){
        this.id = id;
        this.name = name;
        this.email = email;
        this.phoneNumber = phoneNumber;
        this.password = password;
        this.position = position;
        this.salary = salary;
    }
  }